obspy/io/resp/parser.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+'''
+readRESP.py
+script to read RESP (rdseed -R) files into inventory.Response objects.
+
+:copyright:
+    Lloyd Carothers IRIS/PASSCAL, 2016
+:license:
+    GNU Lesser General Public License, Version 3
+    (http://www.gnu.org/copyleft/lesser.html)
+'''
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import io
+import re
+
+from obspy.io.xseed import (parser, blockette, fields)
+
+DEBUG = False
```

Comment:
Debugging should be made available as a `debug=False` kwarg to the individual (or most used) routines.